### PR TITLE
Changes to issue templates & codeowners

### PR DESCRIPTION
**Pull request by mcortesi**
### Description

Review on Code Owners and Issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/design_task.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+## Description
+
+_Explain what needs to be researched or designed_
+
+## Context
+
+_Explain the context for this. Wether it's an initiave, or anything else. WHY is this important?_
```
Comment (pranaymohan) [inline .github/ISSUE_TEMPLATE/design_task.md:16]:
```suggestion
_Explain the context for this. Whether it's an initiative, or anything else. WHY is this important?_
```

File: .github/ISSUE_TEMPLATE/simple_task.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+_Explain what's the need, what's the desired behaviour_
+
+## Current Behaviour
+
+_What's the current behaviour. If there is one_
+
+## Context
+
+_Explain the context for this. Wether it's an initiave, or anything else. WHY is this important?_
```
Comment (pranaymohan) [inline .github/ISSUE_TEMPLATE/simple_task.md:20]:
```suggestion
_Explain the context for this. Whether it's an initiative, or anything else. WHY is this important?_
```

File: .github/ISSUE_TEMPLATE/simple_task.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Solution
+
+_Please describe (if known); what's the actual change we need to do. Describe the solution, strategy for it, analysis, etc_
+
+ * _If **unkwon** on creation. put "Pending"_
```
Comment (pranaymohan) [inline .github/ISSUE_TEMPLATE/simple_task.md:26]:
```suggestion
 * _If **unknown** on creation, put "Pending"_
```

File: .github/ISSUE_TEMPLATE/chore.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+## Description
+
+_Explain what needs to be done_
+
+## Context
+
+_If it makes sense, explain:_
+ * _why is this important? Rationale for doing this_
+ * _Is this part of a bigger initiatve, theme? Which?_
```
Comment (oneeman) [inline .github/ISSUE_TEMPLATE/chore.md:18]:
```suggestion
 * _Is this part of a bigger initiative, theme? Which?_
```

File: .github/ISSUE_TEMPLATE/issue.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+labels: blockchain, needs-research
+assignees: ''
+
+---
+
+## Description
+
+_Explain what's the need, what's the desired behaviour_
+
+## Current Behaviour
```
Comment (oneeman) [inline .github/ISSUE_TEMPLATE/issue.md:14]:
```suggestion
## Current Behavior
```

Behaviour is the British spelling.  Nothing wrong with that, but in the other files we use "behavior" so better to make it consistent.

File: .github/ISSUE_TEMPLATE/issue.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+about: Feature/Need/Problem complete template
+title: ''
+labels: blockchain, needs-research
+assignees: ''
+
+---
+
+## Description
+
+_Explain what's the need, what's the desired behaviour_
```
Comment (oneeman) [inline .github/ISSUE_TEMPLATE/issue.md:12]:
```suggestion
_Explain what's the need, what's the desired behavior_
```

Spelling to make it consistent (see below)

File: .github/ISSUE_TEMPLATE/issue.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+## Current Behaviour
+
+_[Optional] If this a change to a current behaviour, epxlain what's the current behaviour._
```
Comment (oneeman) [inline .github/ISSUE_TEMPLATE/issue.md:16]:
```suggestion
_[Optional] If this is a change to a current behavior, explain what the current behavior is._
```

**Review by oneeman — APPROVED** [review]
Some tiny changes suggested below.  Overall this looks good, but the real test will be when they're actually being used.

**Review by nategraf — APPROVED** [review]
Looks good to me. This is helpful progress

**Pull request merged**